src/vm/appdomain.hpp

```diff
@@ -2603,48 +2603,13 @@ class AppDomain : public BaseDomain
         return (m_dwFlags & IGNORE_UNHANDLED_EXCEPTIONS);
     }
 
-    void SetPassiveDomain()
-    {
-        LIMITED_METHOD_CONTRACT;
-
-        m_dwFlags |= PASSIVE_DOMAIN;
-    }
-
     BOOL IsPassiveDomain()
```

Comment:
If nobody sets it and it is always false, we can delete it too (and fix the few callers that call it).

Reply:
The flag is set by `SetCompilationDomain()`  which I think is used in crossgen.

I can change all the `IsPassiveDomain()` to `IsCompilationDomain()`

Reply:
Ok, I have missed that. You can leave it as it is, or change all `IsPassiveDomain()` to `IsCompilationDomain()`. Either way is fine with me.